praw/reddit.py

```diff
@@ -645,12 +726,12 @@ def request(
 
         :param method: The HTTP method (e.g., GET, POST, PUT, DELETE).
         :param path: The path to fetch.
-        :param params: The query parameters to add to the request (default:
-            None).
         :param data: Dictionary, bytes, or file-like object to send in the body
             of the request (default: None).
         :param files: Dictionary, filename to file (like) object mapping
             (default: None).
+        :param params: The query parameters to add to the request (default:
```

Comment:
Why is this moved?

Reply:
This wasn't sorted, so I thought it might as well be moved

Reply:
I see. In this case, the order here should be consistent with the order the arguments appear. Ideally everything is sorted, but we cannot change the order of these parameters.